src/lib/core/components/visualizations/implementations/LineplotVisualization.tsx

```diff
@@ -944,11 +944,31 @@ function LineplotViz(props: VisualizationProps<Options>) {
       vizConfig.dependentAxisRange ?? defaultDependentAxisRange,
   };
 
+  // set four useState to handle Banner
+  const [
+    showIndependentLogScaleBanner,
+    setShowIndependentLogScaleBanner,
+  ] = useState(false);
+  const [showBinningBanner, setShowBinningBanner] = useState(false);
+  const [
+    showDependentLogScaleBanner,
+    setShowDependentLogScaleBanner,
+  ] = useState(false);
+  const [showErrorBarBanner, setShowErrorBarBanner] = useState(false);
+
+  // add showBanner prop in this Viz
+  const [showBanner, setShowBanner] = useState(true);
```

Comment:
I don't think this state is needed at all?

In CoreUI's `Banner`, `showBanner` defaults to `true` (the only value we ever set it to here in this component) and `setShowBanner` is only used by `Banner` if `autoHideDuration` is non-null (we don't use that prop here).

I mentioned some unclear behaviours here: https://github.com/VEuPathDB/CoreUI/issues/147

Reply:
Good afternoon, @bobular. Thank you for your detailed reviews and tests. 👍 
As for showBanner/setShowBanner, it was necessary with the previous logic to enable Banner component again after closing it. But with this new logic, it may not be required. I will check it carefully on next Monday (today, Friday, I am off to VEuPathDB).